### PR TITLE
Replaces and adds scanner sprites.

### PR DESCRIPTION
**Pull request by Belsima**
Adds sprites for T-Ray upgrades, reagent scanners, antibody scanners, and mining scanners.
Replaces sprites for Alden scanner, forensic scanners (unused), EFTPOS, and samplers.

imageimage

Ported from /tg/station, Baystation 12, and the CEV-Eris.

### CODE REVIEW

File: code/modules/mining/drilling/scanner.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "ore detector"
 	desc = "A complex device used to locate ore deep underground."
 	icon = 'icons/obj/device.dmi'
-	icon_state = "forensic0-old" //GET A BETTER SPRITE.
+	icon_state = "mining1" //GET A BETTER SPRITE.
```
Comment (Belsima) [inline code/modules/mining/drilling/scanner.dm]: This comment probably shouldn't exist. I'll remove it in a coming commit- awaiting input on the t-ray sprites first.

**Comment by MisterLayne** [comment]
They're beautiful ;-;

**Comment by Anewbe** [comment]
I prefer the old Forensic Scanner, in the off chance we ever return to it.
And I'm not particularly a fan of the T-Ray there.

**Comment by Belsima** [comment]
Which one of the two? Also, the old scanner, for reference, was just a modified librarian PDA.

**Comment by Anewbe** [comment]
t-raya
And I'm fine with that, it looked better.

**Comment by Belsima** [comment]
Forensics sprites reverted, new T-Ray-A image

**Comment by Wickedtemp** [comment]
These look lovely : D

It'll be nice to have some variation.

**Comment by Anewbe** [comment]
Doesn't look like that new Tray sprite is actually in the file.

**Comment by Belsima** [comment]
I forgot to push the commit. My PC is currently bricked. Can't fix.

**Comment by Anewbe** [comment]
Alrighty.

**Comment by Belsima** [comment]
Not able to remove sprites from DMI. Removed the trayu state from the code. Will replace whenever I get a PC.

**Pull request closed without merging**